packages/sit-onyx/src/i18n/index.spec.ts

```diff
@@ -75,11 +75,12 @@ test("should translate with/without placeholders", () => {
   message = t.value("does.not.exist" as TestTranslationKey);
   expect(message).toBe("");
 
-  // shows the original placeholders if no values were provided
+  // removes the original placeholders if no values were provided (same behavior as "vue-i18n")
   message = t.value("placeholder" as TestTranslationKey, {
     firstName: undefined,
+    lastName: undefined,
```

Comment:
I left out lastName in this example intentionally because this is what could happen and it should behave still the same?